recipes/baumwelch/build.sh

```diff
@@ -0,0 +1,3 @@
+./configure --prefix="${PREFIX}" 
+make -j"${CPU_COUNT}"
+make -j"${CPU_COUNT}" install
```

Comment:
Is there a `make check`?

Reply:
The checks are no-op at present. (Speaking as upstream maintainer---there are many tests but they are proprietary to upstream and not distributable.)